var/spack/repos/builtin/packages/nvhpc/package.py

```diff
@@ -291,8 +305,8 @@ class Nvhpc(Package):
 
     homepage = "https://developer.nvidia.com/hpc-sdk"
 
-    maintainers("samcmill")
-    tags = ["e4s"]
+    maintainers = ["samcmill"]
+    tag = ["e4s"]
```

Comment:
Why this change? Can you please revert these 2 lines?

Reply:
Done.